diagnostic.md

```diff
@@ -7,36 +7,45 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:kiraamaa/git-basics-diagnostic.git
+
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+Change directory to move into the the cloned directory. Then create a new branch (usually named response) and check it out.
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+Create branch: git branch response
+Switch to branch: git checkout response
+
+You could also use one command of git checkout -b response to make a branch and check it out in sequence.
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+I'm a little confused about the way the question is worded so here are a few answers:
+
+The command ls shows the files that have been changed and that exist within the working directory.
+The command cat, followed by the name of the file containing the new code, shows the full text transcript of the code, which you could examine to see your changes.
+The command diff --, followed by the name of the file containing the new code, shows the deletions and insertions made to an existing file, and thus could be used as a summary of all changes to the code.
+
```

Comment:
"ls" lists all the items in that directory, it would not show you the files that have been changed. 
"cat" would show you exactly what's in file, but it would not show you what changes have been made. 
"git diff" could work, but the most efficient solution would be simply "git status"